pkg/windows/installer/Salt-Minion-Setup.nsi

```diff
@@ -13,18 +13,25 @@
 !define PRODUCT_UNINST_ROOT_KEY "HKLM"
 !define OUTFILE "Salt-Minion-${PRODUCT_VERSION}-Py${PYTHON_VERSION}-${CPUARCH}-Setup.exe"
 
```

Comment:
Please add a variable instead for `"$APPDATA\Salt Project\Salt\"` or for `Salt Project\Salt\`  as it appears in lots of sections of this file.